### ISSUE TITLE
Some system monospace fonts are not recognized for kOS terminal

### ISSUE THREAD

**Issue by kozytskyiad** [KSP-KOS/KOS#3092]
I want the DOS look of the screenshots in the docs but none of the monospaced VGA lookalikes are listed in the font selection menu or under `list fonts.` At the same time Fira Code, a non-standard Windows monospace font, is still listed.

I tried it with the following fonts:
IBM BIOS/VGA 8x8 and 8x16
https://int10h.org/oldschool-pc-fonts/fontlist/

C64 TrueType, after reading that the old font was based off of C64 and specifically installing the "Mono" version
https://style64.org/c64-truetype

Modern DOS
https://www.dafont.com/modern-dos.font

Is there a link to an old guaranteed kOS-compatible retro ish font file maybe?

**Comment by nuggreat** [comment]
Going to ask the obvious question but did you restart KSP after installing the fonts because kOS only looks for new fonts on KSP start.

**Comment by kozytskyiad** [comment]
Yes I did, and I rebooted the computer also just to be sure. Still not showing up.

**Comment by mdpkh** [comment]
I made aspect-ratio-corrected versions of the EGA 9×14 and VGA 9×16 fonts from int10h.org and they work in the kOS terminal for me. I haven't bothered to upload them anywhere, but I could stick them somewhere on my webspace…
